deploy/examples/external/import-external-cluster.sh

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env -S bash
```

Comment:
This script looks duplicated from the parent directory. Can we use a symlink for this too?

Reply:
Here I have changed the Namespace var from` rook-ceph-external` to rook-ceph and also two other variables. I don't think it needs symlink.
Line 7: `NAMESPACE=${NAMESPACE:="rook-ceph"}`
Line17:` ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE"-external"`
Line 28, 29: 
```
CLUSTER_ID_RBD=$NAMESPACE"-external"
CLUSTER_ID_CEPHFS=$NAMESPACE"-external"
```

Reply:
@travisn @smoshiur1237 If we are changing cephcluster to run on  `rook-ceph` instead of `rook-ceph-external` namespace, then we don't need to have separate/duplicate examples yamls or folders.

My main concern for the issue was if we want to support cephcluster  in separate namespaces than rook operator.

So maybe with this change we can just change the default examples with namespace and maybe have a separate folder for all the external related yamls, or lets discuss if we think other way around for keeping support of two cephcluster

Reply:
> Here I have changed the Namespace var from` rook-ceph-external` to `rook-ceph` and also two other variables. If I do not need to change the namespace to `rook-ceph`, I can create the symlink. Otherwise, I do not think it needs symlink. Line 7: `NAMESPACE=${NAMESPACE:="rook-ceph"}` Line17:` ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE"-external"` Line 28, 29:
> 
> ```
> CLUSTER_ID_RBD=$NAMESPACE"-external"
> CLUSTER_ID_CEPHFS=$NAMESPACE"-external"
> ```

If these are just different parameters, couldn't we document to run the same script with the different parameters? Having a single script that allows customized parameters works great to avoid duplication.